.travis.yml

```diff
@@ -11,4 +11,30 @@ sudo: required
 addons:
   chrome: stable
 
-script: "cd $TEST_DIR && npm install && npm test"
+script: "cd ${TRAVIS_BUILD_DIR}/${TEST_DIR} && npm install && npm test"
+
+env:
+  global:
+    # an encrypted 'GITHUB_TOKEN=<token>' for the PolymerLabs/arcs repo
```

Comment:
for the arcs, or arcs-live?

Reply:
It's for arcs - the repo that the build is run inside of needs to be able to decrypt it.